src/assets/schemas/DataRegistrationV1Validation.js

```diff
@@ -1 +1,2176 @@
+            }
+            errors++;
+          }
+        }
+      }
+      else {
+        const err41 = { instancePath: instancePath + "/diseaseSpecificUse", schemaPath: "#/properties/diseaseSpecificUse/type", keyword: "type", params: { type: "array" }, message: "must be array" };
+        if (vErrors === null) {
+          vErrors = [err41];
+        }
+        else {
+          vErrors.push(err41);
+        }
+        errors++;
+      }
+    }
+    if (data.poa !== undefined) {
+      if (typeof data.poa !== "boolean") {
+        const err42 = { instancePath: instancePath + "/poa", schemaPath: "#/properties/poa/type", keyword: "type", params: { type: "boolean" }, message: "must be boolean" };
+        if (vErrors === null) {
+          vErrors = [err42];
+        }
+        else {
+          vErrors.push(err42);
+        }
+        errors++;
+      }
+    }
+    if (data.otherPrimary !== undefined) {
+      if (typeof data.otherPrimary !== "string") {
+        const err43 = { instancePath: instancePath + "/otherPrimary", schemaPath: "#/properties/otherPrimary/type", keyword: "type", params: { type: "string" }, message: "must be string" };
+        if (vErrors === null) {
+          vErrors = [err43];
+        }
+        else {
+          vErrors.push(err43);
+        }
+        errors++;
+      }
+    }
+    if (data.dataLocation !== undefined) {
+      let data28 = data.dataLocation;
+      if (typeof data28 !== "string") {
+        const err44 = { instancePath: instancePath + "/dataLocation", schemaPath: "#/properties/dataLocation/type", keyword: "type", params: { type: "string" }, message: "must be string" };
+        if (vErrors === null) {
+          vErrors = [err44];
+        }
+        else {
+          vErrors.push(err44);
+        }
+        errors++;
+      }
+      if (!((((data28 === "AnVIL Workspace") || (data28 === "Terra Workspace")) || (data28 === "TDR Location")) || (data28 === "Not Determined"))) {
+        const err45 = { instancePath: instancePath + "/dataLocation", schemaPath: "#/properties/dataLocation/enum", keyword: "enum", params: { allowedValues: schema35.properties.dataLocation.enum }, message: "must be equal to one of the allowed values" };
+        if (vErrors === null) {
+          vErrors = [err45];
+        }
+        else {
+          vErrors.push(err45);
+        }
+        errors++;
+      }
+    }
+    if (data.url !== undefined) {
+      let data29 = data.url;
+      if (typeof data29 === "string") {
+        if (func3(data29) < 1) {
+          const err46 = { instancePath: instancePath + "/url", schemaPath: "#/properties/url/minLength", keyword: "minLength", params: { limit: 1 }, message: "must NOT have fewer than 1 characters" };
+          if (vErrors === null) {
+            vErrors = [err46];
+          }
+          else {
+            vErrors.push(err46);
+          }
+          errors++;
+        }
+        // Default ajv URI validation is failing with "TypeError: formats12 is not a function"
```

Comment:
@hams7504 is this the section you commented out?  I found it by searching for "//" in the diff.

Reply:
Yes, that's right! I added line 776 to clarify why I commented out this section.